packages/ts-migrate-plugins/tests/src/explicit-any.test.ts

```diff
@@ -31,78 +31,37 @@ function Foo({
 }
 const { varA, varB: {
   inVarA, inVarB,
-} = {} } = state;
-const { deepobjA = {
-  deepobjB = { deepobjC = {} } = {},
 } = {} } = {};
```

Comment:
This does not cause TS2525, and I'm not totally clear on what it's intending to test.

[TS playground link](https://www.typescriptlang.org/play?#code/MYewdgzgLgBA3gKBjAJgUzQBxAIwFYCCMAvDABSLLLpa54BCJ5cqG2+Awk3AL4x+leASgA0Sftx5CEPAFwwAhmACekgNxA)